src/device_info.sh

```diff
@@ -596,6 +600,28 @@ function get_img_info()
   device_info_data['img_type']="$img_type"
 }
 
+# This function shows displays information
+function get_display_info()
+{
+  local act_name
+  local act_resol
+  local n_display
+  local n_active
+
+  n_display=$(xrandr -q | wc -l)
+
+  n_active=$(xrandr -q | grep -c " connected ")
+
+  act_name=$(xrandr | grep " connected " | awk '{ print $1 }')
+  act_resol=$(xrandr | grep "\*" | awk '{ print $1 }')
+  name_resol=$(xrandr | grep " connected " | awk '{ print $1, "- resolution", $4 }')
```

Comment:
Unfortunately, this will not fly. `xrandr` is not portable across different UIs and needs some extra setup to work via SSH. The best way to handle displays is via the debugfs interface. In this sense, I can see two options:
1. To get this information, Use the `src/plugins/subsystem/drm.sh` file.
2. Just give up on this last commitment and focus.